src/helpers/daily-data-types/combined.tsx

```diff
@@ -5,28 +5,27 @@ import React from "react";
 import { defaultFormatter, heartRateFormatter, minutesFormatter, sleepYAxisConverter } from "./formatters";
 import combinedRestingHeartRate from "../daily-data-providers/combined-resting-heart-rate";
 import { combinedMindfulMinutesDataProvider, combinedSleepDataProvider, combinedStepsDataProvider, combinedTherapyMinutesDataProvider } from "../daily-data-providers";
-import { simpleAvailabilityCheck } from "./availability-check";
-import MyDataHelps from "@careevolution/mydatahelps-js";
+import { checkSourceAvailability, simpleAvailabilityCheck } from "./availability-check";
+import MyDataHelps, { DeviceDataNamespace, DeviceDataV2Namespace } from "@careevolution/mydatahelps-js";
 import { formatNumberForLocale } from "../../helpers/locale";
+import { DataCollectionSettings } from "../daily-data-providers/data-collection-settings";
 
 let combinedTypeDefinitions: DailyDataTypeDefinition[] = [
     {
         dataSource: "Unified",
         type: DailyDataType.RestingHeartRate,
         dataProvider: combinedRestingHeartRate,
         availabilityCheck: async (modifiedAfter?: Date): Promise<boolean> => {
```

Comment:
Outside this PR, but we should add a follow up issue to refactor availabilityCheck to take in DataCollectionSettings; this is gonna be super chatty.